src/awx_plugins/interfaces/_temporary_private_container_api.py

```diff
@@ -39,15 +43,18 @@ def get_incontainer_path(
     if not os.path.isabs(private_data_dir):
         raise RuntimeError('The private_data_dir path must be absolute')
 
-    container_root = pathlib.Path(CONTAINER_ROOT)
+    container_root_path = pathlib.Path(container_root or CONTAINER_ROOT)
```

Comment:
Did you decide against defaulting to `pathlib.Path(CONTAINER_ROOT)`?
If `None` is used as a sentinel, the correct check would be
```suggestion
    container_root_path = pathlib.Path(CONTAINER_ROOT if container_root is None else container_root)
```